src/test/java/no/hvl/dat250/h2020/group5/controllers/PollControllerTest.java

```diff
@@ -0,0 +1,188 @@
+package no.hvl.dat250.h2020.group5.controllers;
+
+import no.hvl.dat250.h2020.group5.repositories.GuestRepository;
+import no.hvl.dat250.h2020.group5.repositories.PollRepository;
+import no.hvl.dat250.h2020.group5.repositories.UserRepository;
+import no.hvl.dat250.h2020.group5.repositories.VoteRepository;
+import no.hvl.dat250.h2020.group5.entities.*;
+import no.hvl.dat250.h2020.group5.enums.AnswerType;
+import no.hvl.dat250.h2020.group5.enums.PollVisibilityType;
+import no.hvl.dat250.h2020.group5.responses.PollResponse;
+import no.hvl.dat250.h2020.group5.responses.VotesResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+
+import java.net.URL;
+import java.util.List;
+import java.util.Objects;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class PollControllerTest {
+
+    @LocalServerPort private int port;
+
+    @Autowired TestRestTemplate template;
+
+    @Autowired PollController PollController;
+
+    @Autowired PollRepository pollRepository;
+
+    @Autowired UserRepository userRepository;
+
+    @Autowired GuestRepository guestRepository;
+
+    @Autowired VoteRepository voteRepository;
+
+    private URL base;
+    private User user1;
+    private Poll poll1;
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        pollRepository.deleteAll();
+        userRepository.deleteAll();
+        guestRepository.deleteAll();
+
+        user1 = new User();
+        user1.setUsername("Admin");
+        user1.setIsAdmin(true);
```

Comment:
Hvis du vil så kan dette bli forenklet til 
```suggestion
        user1 = new User().userName("Admin").admin(true);
```

:)